README.md

```diff
@@ -1,2 +1,73 @@
-# portfolio1
-Second Microverse Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
```

Comment:
- [ ] Kindly add a description of your project to your README, so your README is documented professionally.

Reply:
- [ ] [REQUIRED] You have implemented the project title, kindly also add the project description as provided in the [README Template](https://github.com/microverseinc/readme-template)